.env.default

```diff
@@ -17,7 +17,6 @@ DYNAMIC_TRANSLATION=1
 GEOADMINHOST=mf-geoadmin3.dev.bgdi.ch
 GEODATA_STAGING=test
 HOST=sys-api3.dev.bgdi.ch
-HTTP_PROXY=http://ec2-52-28-118-239.eu-central-1.compute.amazonaws.com:80
```

Comment:
Check with the `infra` if the host still does exist? If so, we may considere ditching it.

Reply:
Good point @ltrod @samfriedli do you know if this host `http://ec2-52-28-118-239.eu-central-1.compute.amazonaws.com` still exists ?

Reply:
Yes this host still exists. The instance is located the aws-admin account in Frankfurt (the only instance in Frankfurt in the aws-admin account)

Reply:
Ok It might be a good idea to have a reminder to clean up this instance, but I guess at best after the migration. I'll created a jira ticket for this task, see https://jira.swisstopo.ch/browse/IGI_SB-1531